Makefile

```diff
@@ -516,7 +516,7 @@ install-docker-buildx: ## Create `docker buildx` builder.
 
 
 .PHONY: golangci
-golangci: GOLANGCILINT_VERSION = v1.51.2
```

Comment:
The version v1.51.2 is available on releases: https://github.com/golangci/golangci-lint/releases/tag/v1.51.2

Can you enable debug logs by adding -d flag?

Reply:
OK, I retried several times and found it's available. Thanks.